app/controllers/recipes_controller.rb

```diff
@@ -19,15 +20,14 @@ def edit; end
 
   # POST /recipes or /recipes.json
   def create
-    @recipe = Recipe.new(recipe_params)
+    @recipe = Recipe.new(params.require(:recipe).permit(:name, :preparation_time, :cooking_time, :description, :public))
```

Comment:
Hi @MattGomb ! great job so far! For this line, you could use the private method we have defined in line 64

Reply:
done!